pills/17/config-nix.txt

```diff
@@ -1,5 +1 @@
-{
-  packageOverrides = pkgs: {
-    graphviz = pkgs.graphviz.override { withXorg = false; };
-  };
-}
+{ packageOverrides = pkgs: { graphviz = pkgs.graphviz.override { withXorg = false; }; }; }
```

Comment:
That looks so weird... I don't like so much this change TBH.

Reply:
Unfortunately, this is what the formatter gives us and I'm not a fan of manually fixing the formatted output.
So I added a comment into the innermost attrset which forces an expansion (which I think is the issue you had)

Reply:
That's just perfect now. Thank you !